### PR TITLE
Nodejs week2

### PR DESCRIPTION
**Pull request by danJecu**
NodeJS-Week2-Homework

### CODE REVIEW

File: nodejs/week2/app.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+app.get('/documents/:id', (req, res) => {
+    const id = req.params.id;
+
+    const matchingId = data.filter(item => item.id === parseInt(id));
+
+    if (!matchingId.length) {
+        res.status(404).json('error: Not Found');
+    } else {
+        res.send(matchingId);
+    }
```
Comment (Divyajg) [inline nodejs/week2/app.js:37]: The name `matchingId `says it is an id, not the document. You can call it `documentWithId ` or something like that.
Variable names should descriptive.

File: nodejs/week2/app.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+app.use(express.json());
+
+app.get('/', (req, res) => {
+    res.send('This is a search engine');
+});
+
+app.get('/search', (req, res) => {
+    const queryValue = req.query.q;
+    if (queryValue) {
+        const matchingQuerry = data.filter(item => {
```
Comment (Divyajg) [inline nodejs/week2/app.js:16]: Double-check for spelling, I know it is very minor but be careful. And also the name should be descriptive.

**Review by Divyajg — APPROVED** [review]
Nice work.
I left a small suggestion in the comments.

File: nodejs/week2/app.js (hunk 1 above)
Comment (danJecu) [reply nodejs/week2/app.js:37]: It was fixed.

File: nodejs/week2/app.js (hunk 2 above)
Comment (danJecu) [reply nodejs/week2/app.js]: I made the name more descriptive.

Thank you for your time!

**Comment by danJecu** [comment]
Thank you!

**Pull request closed without merging**